### PR TITLE
Contributing page updated with helm chart tests suit

### PR DESCRIPTION
**Pull request by Unknovvn**
Changes according to [ticket](https://github.com/elastic/cloud-on-k8s/issues/6729)

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)? ✅ 
- Have you followed the [contributor guidelines](https://github.com/elastic/cloud-on-k8s/tree/main/CONTRIBUTING.md)? ✅ 
- If you submit code, is your pull request against main? We recommend pull requests against main. We will backport them as needed. ✅

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -48,19 +48,21 @@ Your contributions should pass the existing tests. You must provide new tests to
...
+  They use the standard `go test` tooling. See the `test/e2e` directory. We recommend to rely primarily on unit and integration tests, as e2e tests are slow and hard to debug because they simulate real user scenarios. To run a specific e2e test, you can use something similar to `make TESTS_MATCH=TestMetricbeatStackMonitoringRecipe clean docker-build docker-push e2e-docker-build e2e-docker-push e2e-run`. This will run the e2e test with your latest commit and is very close to how it will run in CI.
 
   A faster option is to run the operator and tests locally, with `make run` in one shell and `make e2e-local TESTS_MATCH= TestMetricbeatStackMonitoringRecipe` in another, though this does not exercise all of the same configuration (permissions etc.) that will be used in CI, so is not as thorough.
-  
+
+- **Helm chart tests** - allows us to test helm charts. To run helm chart tests, you can use `make helm-test` command. This will run test script.
```
Comment (naemono) [inline CONTRIBUTING.md:64]:
```suggestion
- **Helm chart tests** - allows us to test helm charts. To run helm chart tests, you can use `make helm-test` command.
```

**Review by naemono — COMMENTED** [review]
Mostly LGTM. Would love a member of @elastic/docs to take a look as well.

**Comment by naemono** [comment]
@elasticmachine run elasticsearch-ci/docs

**Comment by naemono** [comment]
closes #6729

**Comment by barkbay** [comment]
buildkite test this

**Pull request merged**